src/frontend/src/lib/components/convert/ConvertValue.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	export let ref: undefined | string = undefined;
+</script>
+
+<div class="mb-2 flex w-full justify-between last:mb-0 md:items-center">
+	<label for={ref} class="mr-1 text-sm sm:mr-2 sm:text-base"><slot name="label" /></label>
+
+	<div class="flex flex-col items-end sm:flex-row sm:items-center">
+		<span
+			class={`mb-1 text-sm font-bold sm:mb-0 sm:text-base ${$$slots['secondary-value'] ? 'mr-0 sm:mr-3' : ''}`}
```

Comment:
not a call to aciotn, but usually, in the other components, we put the $$slots outside:

```typescript
	let amount = true;
	$: amount = nonNullish($$slots.amount);
```
	
just to keep better track

don't know if useful

Reply:
Nice, updated it.